web/scripts/frame.js

```diff
@@ -57,6 +62,21 @@ removeFirebase = function () {
     removeScript('firestore');
 }
 
+removeCanvaskit = function () {
+    var scripts = document.head.querySelectorAll('script');
+    var existingScript;
+    for (var i = 0; i < scripts.length; i++) {
+        if (scripts[i].src.includes('canvaskit.js')) {
+            existingScript = scripts[i];
```

Comment:
Can you just move your `existingScript.parentNode.removeChild(existingScript);` logic here since you're only acting on the first one you find anyway?

```js
for (var i = 0; i < scripts.length; i++) {
    if (scripts[i].src.includes('canvaskit.js')) {
        scripts[i].parentNode.removeChild(existingScript);
        break;
    }
}
```